hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -226,6 +229,31 @@ object HoodieSparkUtils extends SparkAdapterSupport with SparkVersionsSupport wi
     sparkAdapter.createSparkRowSerDe(structType)
   }
 
+  def doParsePartitionColumnValues(partitionColumns: Array[String],
```

Comment:
IMO, we should rename this method to `parsePartitionColumnValues` and `parsePartitionColumnValues` to `doParsePartitionColumnValues`

Reply:
Agree with you, current naming is confusing. Addressed your comment in [7ee1845](https://github.com/apache/hudi/pull/12192/commits/7ee1845ce0c56dbad6216646b565618223ad38a1)